sysconfig/bbpv/tools/packages.yaml

```diff
@@ -167,6 +171,6 @@ packages:
         buildable: False
         version: [cluster.2018.1]
     all:
-        compiler: [gcc, intel]
+        compiler: [gcc@4.8.5, gcc@5.4.0, intel]
```

Comment:
just wondering, why gcc@5.4.0 ? seems like an odd version to mention here for bbp5

Reply:
I kept 4.8.5 as default because it system one. Added 5.4.0 which is required for HPCToolkit (for C++11 support). We can use higher version of gcc but I haven't tested latest gcc for Hpctoolkit.

We will create use separate config for users. This is more for tools installation where we can use older gcc versions (to avoid libc issues).

Reply:
Ah, thanks for the explanation. Makes sense then.